docs/Debugging.md

```diff
@@ -96,43 +96,83 @@ The debugger will receive a list of all project roots, separated by a space. For
 
 ### Debugging with [Stetho](http://facebook.github.io/stetho/) on Android
 
+Following this guide to enable Stetho only for Debug mode:
+
 1. In ```android/app/build.gradle```, add these lines in the `dependencies` section:
 
    ```gradle
-   compile 'com.facebook.stetho:stetho:1.3.1'
-   compile 'com.facebook.stetho:stetho-okhttp3:1.3.1'
+    debugCompile 'com.facebook.stetho:stetho:1.x.x'
+    debugCompile 'com.facebook.stetho:stetho-okhttp3:1.x.x'
    ```
 
-2. In ```android/app/src/main/java/com/{yourAppName}/MainApplication.java```, add the following imports:
-
-   ```java
-   import com.facebook.react.modules.network.ReactCookieJarContainer;
-   import com.facebook.stetho.Stetho;
-   import okhttp3.OkHttpClient;
-   import com.facebook.react.modules.network.OkHttpClientProvider;
-   import com.facebook.stetho.okhttp3.StethoInterceptor;
-   import java.util.concurrent.TimeUnit;
+Go to http://facebook.github.io/stetho/ and replace 1.x.x with the latest version. For example:
+   
+   ```gradle
+    debugCompile 'com.facebook.stetho:stetho:1.4.2'
+    debugCompile 'com.facebook.stetho:stetho-okhttp3:1.4.2'
```

Comment:
Could remove this bit and specify the version to use.
The version is dependant on the version of OkHttp React native is compatible is.